config/stylelint/sassMsgFormatter.js

```diff
@@ -27,12 +27,12 @@ function filterForErrors(result) {
 function filterDuplicates(results) {
   let filteredArray = [];
   let isUnique = true;
-  results.forEach(result => {
+  results.forEach((result) => {
     if (filteredArray.length == 0) {
       // if the filtered array is empty then it's automatically unique
       filteredArray.push(result);
     } else {
-      filteredArray.forEach(uniqueResult => {
+      filteredArray.forEach((uniqueResult) => {
```

Comment:
Do we want the added parens in our code base? Not opposed I suppose.

Reply:
[The blog announcement for Prettier v2](https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-arrowparens-to-always-7430httpsgithubcomprettierprettierpull7430-by-kachkaevhttpsgithubcomkachkaev) has a good explanation that rings true for me:

> ## Change default value for arrowParens to always
> Since version 1.9, Prettier has had an option to always wrap arrow function arguments with parentheses. In version 2.0, this behavior has become the default.
> ```
> // Input
> const fn = (x) => y => x + y;
> 
> // Prettier 1.19
> const fn = x => y => x + y;
> 
> // Prettier 2.0
> const fn = (x) => (y) => x + y;
> ```
> At first glance, avoiding parentheses in the isolated example above may look like a better choice because it results in less visual noise. However, when Prettier removes parentheses, it becomes harder to add type annotations, extra arguments, default values, or a variety of other things. Consistent use of parentheses provides a better developer experience when editing real codebases, which justifies the change.